backend/infrahub/computed_attribute/triggers.py

```diff
@@ -0,0 +1,70 @@
+from infrahub.trigger.models import BuiltinTriggerDefinition, EventTrigger, ExecuteWorkflow
+from infrahub.workflows.catalogue import (
+    COMPUTED_ATTRIBUTE_REMOVE_PYTHON,
+    COMPUTED_ATTRIBUTE_SETUP,
+    COMPUTED_ATTRIBUTE_SETUP_PYTHON,
+)
+
+TRIGGER_COMPUTED_ATTRIBUTE_PYTHON_SETUP_BRANCH = BuiltinTriggerDefinition(
+    name="computed-attribute-python-setup-on-branch-creation",
+    previous_names={"Trigger-schema-update-event"},
+    description="Trigger actions on branch create event",
+    trigger=EventTrigger(events={"infrahub.branch.created"}),
+    actions=[
+        ExecuteWorkflow(
+            name=COMPUTED_ATTRIBUTE_SETUP_PYTHON.name,
+            parameters={
+                "branch_name": "{{ event.resource['infrahub.branch.name'] }}",
+                "trigger_updates": False,
+            },
+        )
+    ],
+)
+
+TRIGGER_COMPUTED_ATTRIBUTE_PYTHON_SETUP_COMMIT = BuiltinTriggerDefinition(
+    name="computed-attribute-python-setup-on-commit",
+    description="Trigger actions on branch create event",
+    trigger=EventTrigger(events={"infrahub.repository.update_commit"}),
+    actions=[
+        ExecuteWorkflow(
+            name=COMPUTED_ATTRIBUTE_SETUP_PYTHON.name,
+            parameters={
+                "branch_name": "{{ event.resource['infrahub.branch.name'] }}",
+                "commit": "{{ event.payload['commit'] }}",
+            },
+        )
+    ],
+)
+
+TRIGGER_COMPUTED_ATTRIBUTE_PYTHON_CLEAN_BRANCH = BuiltinTriggerDefinition(
+    name="computed-attribute-python-cleanup-on-branch-deletion",
+    description="Trigger actions on branch delete event",
+    trigger=EventTrigger(events={"infrahub.branch.deleted"}),
+    actions=[
+        ExecuteWorkflow(
+            name=COMPUTED_ATTRIBUTE_REMOVE_PYTHON.name,
+            parameters={
+                "branch_name": "{{ event.resource['infrahub.branch.name'] }}",
+            },
+        )
+    ],
+)
+
+TRIGGER_COMPUTED_ATTRIBUTE_ALL_SCHEMA = BuiltinTriggerDefinition(
+    name="computed-attribute-all-setup-on-schema-update",
+    trigger=EventTrigger(events={"infrahub.schema.update"}),
+    actions=[
+        ExecuteWorkflow(
+            name=COMPUTED_ATTRIBUTE_SETUP.name,
+            parameters={
+                "branch_name": "{{ event.resource['infrahub.branch.name'] }}",
+            },
+        ),
+        ExecuteWorkflow(
+            name=COMPUTED_ATTRIBUTE_SETUP_PYTHON.name,
+            parameters={
+                "branch_name": "{{ event.resource['infrahub.branch.name'] }}",
+            },
+        ),
+    ],
+)
```

Comment:
With these changes it would be nice if we could have a unittest that parsed the `actions` of these and matched the name of the workflow with the function signature of that workflow to validate if at least the parameter keys were correct. I haven't looked at the internals of `BuiltinTriggerDefinition` to see if we have direct access to the data we need when doing this but it seems reasonable to think that it should be easily accessible.